tests/ci/integration/run_sslproxy_integration.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash -exu
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+
+source tests/ci/common_posix_setup.sh
+
+# Set up environment.
+
+# SYS_ROOT
+#  - SRC_ROOT(aws-lc)
+#    - SCRATCH_FOLDER
+#      - sslproxy
+#      - AWS_LC_BUILD_FOLDER
+#      - AWS_LC_INSTALL_FOLDER
+#      - SSLPROXY_BUILD_FOLDER
+#      - libevent
+#      - LIBEVENT_INSTALL_FOLDER
+
+# Assumes script is executed from the root of aws-lc directory
+SCRATCH_FOLDER=${SRC_ROOT}/"SSLPROXY_BUILD_ROOT"
```

Comment:
I believe the best practice is to have this be `"${SRC_ROOT}/SSLPROXY_BUILD_ROOT"`. (variable should be inside the quotes). There are a few other lines where the two styles are mixed as well.

Reply:
Will align and check the other NGINX PR while I'm at it as well.